docs/source/notebooks/finch-usage.ipynb

```diff
@@ -81,13 +81,15 @@
      "text": [
       "ProcessSucceeded\n",
       "frost_daysResponse(\n",
-      "    output_netcdf='https://pavics.ouranos.ca/wpsoutputs/e8afbb04-42d3-11ea-9531-0242ac12000b/out.nc',\n",
-      "    output_log='https://pavics.ouranos.ca/wpsoutputs/e8afbb04-42d3-11ea-9531-0242ac12000b/log.txt'\n",
+      "    output_netcdf='http://https://pavics.ouranos.ca/wpsoutputs/e8afbb04-42d3-11ea-9531-0242ac12000b/frost-days_SRES-A2-experiment_20460101-20650101.nc',\n",
+      "    output_log='https://pavics.ouranos.ca/wpsoutputs/e8afbb04-42d3-11ea-9531-0242ac12000b/log.txt',\n",
+      "    ref='https://pavics.ouranos.ca/wpsoutputs/e8afbb04-42d3-11ea-9531-0242ac12000b/input.meta4'\n",
       ")\n"
      ]
     }
    ],
    "source": [
+    "# NBVAL_IGNORE_OUTPUT\n",
```

Comment:
Oh you added `NBVAL_IGNORE_OUTPUT` !  Sorry I didn't saw this one, can you move it?